src/NServiceBus.Msmq.AcceptanceTests/When_setting_label_generator.cs

```diff
@@ -62,6 +63,37 @@ public class Context : ScenarioContext
             public bool GeneratorWasCalled { get; set; }
         }
 
+        class StartHandler : Feature
+        {
+            protected override void Setup(FeatureConfigurationContext context)
+            {
+                context.Container.ConfigureComponent<StartHandlerTask>(DependencyLifecycle.SingleInstance);
+                context.RegisterStartupTask(b => b.Build<StartHandlerTask>());
+            }
+        }
+
+        class StartHandlerTask : FeatureStartupTask
+        {
+            Context context;
+            ReadOnlySettings settings;
+
+            public StartHandlerTask(Context context, ReadOnlySettings settings)
+            {
+                this.context = context;
+                this.settings = settings;
+            }
+
+            protected override Task OnStart(IMessageSession session)
+            {
+                context.GeneratorWasCalled = settings.Get<bool>("GeneratorWasCalled");
```

Comment:
looks like GeneratorWasCalled is never used for assertions at first glance. Can we just kill all the Feature code?